### ISSUE TITLE
snuffler crash - right click marker table

### ISSUE THREAD

**Issue by marzeck** [pyrocko/pyrocko#240]
Hi guys,
When having the marker table ('m'-button) open and right-click on one of the listed markers  my snuffler crashes and show the following error message:

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/gui/marker_editor.py", line 310, in contextMenuEvent
    self.right_click_menu.popup(qw.QCursor.pos())
AttributeError: 'module' object has no attribute 'QCursor'
Aborted (core dumped)
```

The problem occurred on Ubuntu 16.04 and with the newest pyrocko version (updated this morning).

**Comment by marzeck** [comment]
Seems to be a Qt5 related issue. With Qt4 everything works fine.
Under Qt5 the function QCursor doesn't exist anymore.

**Comment by HerrMuellerluedenscheid** [comment]
fixed: 82f0544
Please pull an update from `master`
